internal/optimization_checks.py

```diff
@@ -989,6 +989,20 @@ def check_images(self):
                                 check['info']['metadata'] = metadata
                         except Exception:
                             pass
+                        # Use imagemagick to convert metadata to json
+                        try:
+                            command = '{0} -verbose "{1}" json:-'.format(self.job['image_magick']['convert'], request['body'])
+                            subprocess.call(command, shell=True)
+                            magick = json.loads(subprocess.check_output(command, shell=True, encoding='UTF-8'))
+                            if magick is not None:
+                                if 'image' in magick:
+                                    remove = ['name', 'artifacts', 'colormap', 'proprties', 'version']
```

Comment:
Should `proprties` be `properties`? Or is this like how the variable fonts folks decided to abbreviate `weight` to `wght`?

Reply:
Argh.  Good catch, but looking at a few more samples, there can be some interesting details about the compressor used to create the image in there so I decided to leave it after all (well, remove the string which was just wrong, leave `properties`)